concepticondata/conceptlists/Wang-2021-90.tsv

```diff
@@ -0,0 +1,91 @@
+Wang-2021-90-78	78	3970	PROCESS	process	word78	过程	n	2	5	0.03216820339	0.005623670526	0.01365569755	0.002641526973	0.03399032016	0.005463729194	0.03680924169	0.009009102015	0.3644320425	0.06078101398	0.06370646011	-1.193911481	3.884615385	3.333333333	1.107142857	1.2	1.444444444	1.958333333	4	1.861760953	1.860521019	0.5669467095	0.6454972244	0.8473185457	1.267629306	0	-1.19739	-0.95573	1.70511	-1.97073	18	6.307692308	2.409933123	3.560310857	3.293512443	4.453003927	4.196937208	-1.55672	-0.51745	26.84095212	16.27848611	35.30317859
+Wang-2021-90-79	79	3971	METHOD	method	word79	方法	n	2	5	0.02280721649	0.005521478726	0.01051231464	0.002817098731	0.02380203119	0.005165163276	0.02750312457	0.008938236867	0.3097742148	0.07074013094	0.05385590417	-1.033294891	3.846153846	3.777777778	1.071428571	1.4	1.481481481	1.875	4.074074074	1.433661262	1.948043068	0.377964473	1.118033989	1.25177936	1.261899879	0.3849001795	-1.07905	-0.98252	1.21862	-1.03797	12	6.423076923	2.245512668	3.109192403	3.06578673	3.201008489	4.014253101	-1.41439	-0.99198	19.94450464	10.39779837	28.67417786
+Wang-2021-90-80	80			phenomenon	word80	现象	n	2	5	0.02452128971	0.006268005083	0.01403616811	0.00290090903	0.02809722605	0.006703527124	0.02508404018	0.01006039488	0.2510173821	0.05994096699	0.04151885856	-1.299312078	3.461538462	3.518518519	1.035714286	1.2	1.703703704	2.125	3.962962963	1.303250374	1.783910363	0.1889822365	0.8164965809	1.409168189	1.19099885	0.4369018092	-1.26633	-0.79375	1.32024	-1.07983	19	6.115384615	2.29666519	3.515933714	3.25145332	3.855747025	3.57704562	-1.62046	-1.3369	27.72927512	11.90898384	34.34404192
+Wang-2021-90-81	81	3972	EXCITED	excited	word81	兴奋	a	2	4	0.0307364592	0.007618534017	0.01256238643	0.002588896419	0.03111693469	0.006544291414	0.02782753365	0.008298618729	0.2541805933	0.07613728489	0.0645626578	-0.1120662944	4.769230769	5.111111111	1.035714286	1.28	2.703703704	4.625	5.296296296	1.531213697	1.648620247	0.1889822365	1.021436896	2.034600978	1.610090465	0.8688991627	-0.54045	0.17165	0.95861	0.81762	14	6.461538462	1.770852012	2.6084198	2.477780737	3.56049291	4.006155537	0.23949	0.40874	26.22437072	15.06846375	38.38259117
+Wang-2021-90-82	82	1495	HAPPY	comfortable	word82	舒心	a	2	4	0.0334804302	0.006236274634	0.01774503191	0.003097627411	0.03352057866	0.006253782234	0.03774770369	0.008893270778	0.2729474838	0.05289269009	0.06269500217	-0.8076586059	3.961538462	4.222222222	1.178571429	1.44	1.481481481	3.041666667	5.481481481	1.482720991	1.867399124	0.7723735149	0.916515139	0.9754829325	1.781039429	0.7529618629	-1.01523	-1.10004	0.59998	0.31055	16	6.038461538	0.903089987	2.281052012	3.319993768	2.48641475	4.000051897	-0.91877	-0.34044	32.12494475	15.70832091	44.14936326
```

Comment:
This should be "unmapped", as it does not mean "HAPPY".

Reply:
Do you think it would make sense to add a new encompassing concept COMFORTABLE or AT EASE in this case, or would that be an instance of fuzzy mapping, which is to be avoided?